apps/meteor/app/slackbridge/server/RocketAdapter.js

```diff
@@ -264,15 +265,15 @@ export default class RocketAdapter {
 
 		for await (const slack of this.slackAdapters) {
 			if (addedRoom) {
-				return;
+				continue;
```

Comment:
I believe we could use a `break` here, couldn't we?
Since once this variable is set, it won't be undefined ever again

Reply:
I still think this could be a `break`, but we can leave this for a refactor

Reply:
yeah, should be a break. But since we're refactoring this soon, it may get removed there with other logic changes i hope.